core/scripts/local/DeployEMP.js

```diff
@@ -59,13 +59,13 @@ const deployEMP = async callback => {
       expirationTimestamp: "1596240000", // 2020-08-01T00:00:00.000Z. Note, this date will no longer work once it is in the past.
       collateralAddress: collateralToken.address,
       priceFeedIdentifier: priceFeedIdentifier,
-      syntheticName: "ETH/BTC",
-      syntheticSymbol: "ETH/BTC",
+      syntheticName: "ETH/BTC Synthetic Token Expiring 1 August 2020",
+      syntheticSymbol: "ETHBTC-Aug20",
```

Comment:
nit: Why not capitalize `Aug` to `AUG`?

Reply:
ye, I agree with all caps for the symbol. looks better all caps IMHO.